samcli/commands/_utils/options.py

```diff
@@ -849,8 +849,10 @@ def resolve_image_repos_option(f):
 
 def use_container_build_click_option():
     return click.option(
-        "--use-container",
+        "--use-container/--no-use-container",
```

Comment:
If customer doesn't specify `--use-container` or `--no-use-container`, will sam cli still use the value from samconfig.toml after this change?

Reply:
This documentation might help answer your question: [SAM CLI Configuration File Precedence](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html#serverless-sam-cli-config-precedence).

Reply:
If the current samconfig.toml file has `use-container = True` and customer run sam command with `--no-use-container --save-parameters`, will we remove the `use-container = True` from samconfig.toml file before saving `--no-use-container = True`?